.travis.yml

```diff
@@ -24,10 +24,10 @@ cache:
 
 before_install: . ./ci/before_install.sh
 
+before_script: cd libbindgen
+
 script:
-  - cargo build --verbose --features "$BINDGEN_FEATURES"
   - cargo test --features "$BINDGEN_FEATURES"
-  - cargo build --release --verbose --features "$BINDGEN_FEATURES"
   - cargo test --release --features "$BINDGEN_FEATURES"
```

Comment:
It seems travis will no longer build bindgen executable.
Would it make sense to not cd in libbindgen and do something along the line of(if that works):

/// Build library and tests:
cargo test -p libbindgen --features "$BINDGEN_FEATURES"
cargo test  -p libbindgen --release --features "$BINDGEN_FEATURES"
cargo build -p libbindgen --features "$BINDGEN_FEATURES _docs"

/// build executable and test (even though we don't really have any test for the executable)
cargo test --features "$BINDGEN_FEATURES"
cargo build --features "$BINDGEN_FEATURES _docs"

/// Verify the expectations build
cargo test -p tests_expectations      <<<<- still need to describe this as a dev dependency of bidgen cargo.toml